src/main/java/de/teamlapen/vampirism/tileentity/TileTotem.java

```diff
@@ -392,7 +392,10 @@ public void onTileRemoved() {
         if (v != null) {
             v.removeTotemAndReset(this.pos);
         }
-        updateBossinfoPlayers(null);
+        if (capturingFaction != null)
```

Comment:
Pls only skip the curly brackets for one liners. With if/else it might be missed at some point